conversion-measurement/sites/home/shared.browser-setup.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const MIN_SUPPORTED_BROWSER_VERSION = 105
+const SUPPORTED_USER_AGENTS = ['Google Chrome']
```

Comment:
I would prefer that we also include `Chromium`, as otherwise it will be more difficult for us to use the demo with Chromium built from source.